tests/actions/ReportTest.js

```diff
@@ -13,7 +13,30 @@ PushNotification.register = () => {};
 PushNotification.deregister = () => {};
 
 describe('actions/Report', () => {
-    it('should store a new report action in Onyx when one is handled via Pusher', () => {
+    beforeAll(() => {
+        // When using the Pusher mock the act of calling Pusher.isSubscribed will create a
+        // channel already in a subscribed state. These methods are normally used to prevent
+        // duplicated subscriptions, but we don't need them for this test so forcing them to
+        // return false will make the testing less complex.
+        Pusher.isSubscribed = jest.fn().mockReturnValue(false);
+        Pusher.isAlreadySubscribing = jest.fn().mockReturnValue(false);
+
+        // Connect to Pusher
+        PusherConnectionManager.init();
+        Pusher.init({
+            appKey: CONFIG.PUSHER.APP_KEY,
+            cluster: CONFIG.PUSHER.CLUSTER,
+            authEndpoint: `${CONFIG.EXPENSIFY.URL_API_ROOT}api?command=Push_Authenticate`,
+        });
+    });
+
+    afterEach(() => {
+        // Unsubscribe from account channel after each test since we subscribe in the function
+        // subscribeToUserEvents and we don't want duplicate event subscriptions.
+        Pusher.unsubscribe('private-user-accountID-1');
```

Comment:
Should this have the user's accountID? Or is this supposed to be hard-coded?

Reply:
Oh is it supposed to be like this because this is a test?

Reply:
Yeah it's just hard-coded to 1 because it's a test, and I wanted to be able to unsubscribe in one place instead of doing `Pusher.unsubscribe(`private-user-accountID-${TEST_USER_ACCOUNT_ID}`);` at the end of both tests.

Other possible solutions are:
1. Move `const TEST_USER_ACCOUNT_ID = 1;` outside of each test to the top of the file, so we can do `Pusher.unsubscribe(`private-user-accountID-${TEST_USER_ACCOUNT_ID}`);` inside the `afterEach` method
2. I can move the `unsubscribe` to the bottom of each test so we can use `Pusher.unsubscribe(`private-user-accountID-${TEST_USER_ACCOUNT_ID}`);`

Either way, it's best to unsubscribe from subscriptions set up during tests so we clean things up :)